R/virtual-select.R

```diff
@@ -168,11 +181,7 @@ virtualSelectInput <- function(inputId,
     tags$div(
       id = inputId,
       class = "virtual-select",
-      style = css(
-        width = "100%",
```

Comment:
Without changing this - the dropdown usually is not in the correct width.  I don't know if this needs to interact with `width` argument (and I'm not sure you can do a `width = "fit"` if there is no `data-width`).